protobuf-codegen/src/main/proto/messages.proto

```diff
@@ -360,3 +361,9 @@ enum LegalHoldStatus {
     DISABLED = 1;
     ENABLED = 2;
 }
+
+enum UnknownStrategy {
+    IGNORE = 0;                 // Ignore the message completely.
```

Comment:
should we leave those comments here?

Reply:
Doesn't hurt to have them :D
They were added just to have a bit of documentation in the .proto files